src/components/VSelect/VSelect.js

```diff
@@ -237,7 +237,10 @@ export default {
         }
       }
     },
-    findExistingItem (item) {
+    findExistingItem (val) {
+      return this.items.find(i => this.valueComparator(this.getValue(i), this.getValue(val)))
```

Comment:
Store `this.getValue(val)` in a variable. It should not be re-evaluated for every find iteration.

Reply:
Fixed in [7819c6f9](https://github.com/rokkie/vuetify/commit/7819c6f960e02deb5d21782b82dd7d027d00214d)